### ISSUE TITLE
aips broken for KERN-dev and KERN-4

### ISSUE THREAD

**Issue by gijzelaerr** [kernsuite/packaging#150]
```
$ /usr/bin/runaips
/usr/bin/runaips: 69: /usr/bin/runaips: aips: not found
```

I think something went wrong with the update.

**Comment by ngizani** [comment]
The same issue is present using Kern-4.

**Comment by gijzelaerr** [comment]
i've uploaded a fixed AIPS 31dec18 to KERN-4 and KERN-dev
